.github/fabricbot.json

```diff
@@ -199,58 +171,16 @@
     {
       "taskType": "trigger",
       "capabilityId": "IssueResponder",
-      "subCapability": "PullRequestResponder",
+      "subCapability": "IssuesOnlyResponder",
```

Comment:
Why IssuesOnlyResponder if this applies to PRs?

Reply:
Interestingly, this task was previously marked as `IssuesOnlyResponder` and the diff display merely makes it look like this was edited. So it was wrong before too, and I'm unsure if the task was previously functioning properly.

I'll go ahead and fix it regardless. Good catch!